scripts/build-image.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Base image
+NODEJS_BASE_IMAGE=nodejs-16
```

Comment:
Can we source the base image from `.aicoe-ci.yaml` instead?

Reply:
Good Idea, changed it WDYT? @tumido

Reply:
I :heart: it @mimotej  Looks great!